content/en/blog/2019/webhook/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Secure webhook management
+description: A more secure way to manage Istio webhooks.
+publishdate: 2019-10-29
+attribution: Lei Tang (Google)
+keywords: [security, kubernetes, webhook]
+target_release: 1.4
+---
+
+Istio has two webhooks: Galley and the sidecar injector. Both of them are crucial for Istio,
+with Galley validating Kubernetes resources and the sidecar injector injecting Istio
+sidecar containers.
+
+By default, Galley and the sidecar injector manage their own webhook configurations, which may
+have a security risk if they are compromised, for example, through buffer overflow attacks. Configuring
+a webhook is a highly privileged operation as a webhook may monitor and mutate all
+Kubernetes resources. In the following example, the attacker compromises
+Galley and modifies the webhook configuration of Galley to eavesdrop all Kubernetes secrets
+(the `clientConfig` is modified by the attacker to direct the `secrets` resources to the
+a service owned by the attacker).
```

Comment:
(the `clientConfig` is modified by the attacker to direct the `secrets` resources to ~the~
a service owned by the attacker).

Reply:
Revised accordingly.